samples/word/10-content-controls/insert-and-change-checkbox-content-control.yaml

```diff
@@ -0,0 +1,246 @@
+              `id: ${selectedContentControl.id} ... isChecked: ${selectedContentControl.checkboxContentControl.isChecked}`
+            );
+          });
+        }
+
+        async function toggleCheckboxContentControls() {
+          // Toggles the isChecked property on all checkbox content controls.
+          await Word.run(async (context) => {
+            let contentControls = context.document.getContentControls({
+              types: [Word.ContentControlType.checkBox]
+            });
+            contentControls.load("items");
+
+            await context.sync();
+
+            const length = contentControls.items.length;
+            console.log(`There are ${length} checkbox content controls.`);
+
+            if (length <= 0) {
+              return;
+            }
+
+            const checkboxContentControls = [];
+            for (let i = 0; i < length; i++) {
+              let contentControl = contentControls.items[i];
+              contentControl.load("id,checkboxContentControl/isChecked");
+              checkboxContentControls.push(contentControl);
+            }
+
+            await context.sync();
+
+            console.log("isChecked state before:");
+            const updatedCheckboxContentControls = [];
+            for (let i = 0; i < checkboxContentControls.length; i++) {
+              const currentCheckboxContentControl = checkboxContentControls[i];
+              const isCheckedBefore = currentCheckboxContentControl.checkboxContentControl.isChecked;
+              console.log(`id: ${currentCheckboxContentControl.id} ... isChecked: ${isCheckedBefore}`);
+
+              currentCheckboxContentControl.checkboxContentControl.isChecked = !isCheckedBefore;
+              currentCheckboxContentControl.load("id,checkboxContentControl/isChecked");
+              updatedCheckboxContentControls.push(currentCheckboxContentControl);
+            }
+
+            await context.sync();
+
+            console.log("isChecked state after:");
+            for (let i = 0; i < updatedCheckboxContentControls.length; i++) {
+              const currentCheckboxContentControl = updatedCheckboxContentControls[i];
+              console.log(
+                `id: ${currentCheckboxContentControl.id} ... isChecked: ${currentCheckboxContentControl.checkboxContentControl.isChecked}`
+              );
+            }
+          });
+        }
+
+        async function deleteCheckboxContentControl() {
+          // Deletes the first checkbox content control found in the selection.
+          await Word.run(async (context) => {
+            const selectedRange = context.document.getSelection();
+            let selectedContentControl = selectedRange
+              .getContentControls({
+                types: [Word.ContentControlType.checkBox]
+              })
+              .getFirstOrNullObject();
+            selectedContentControl.load("id");
+
+            await context.sync();
+
+            if (selectedContentControl.isNullObject) {
```

Comment:
Same comment as with the toggle method - exceptions are thrown when no content controls are selected.

Reply:
Added handling in tryCatch